src/core2/graph.test.js

```diff
@@ -162,8 +172,8 @@ describe("graph", () => {
       });
       describe("filters edges by type:", () => {
         [
-          ["EDGE", [fooNeighbor.bar, fooNeighbor.absent]],
-          ["SELF", [fooNeighbor.foo]],
+          ["SIMPLE", [fooNeighbor.bar, fooNeighbor.absent]],
```

Comment:
Why not use `SimpleEdge.TYPE` here (and create that accordingly), as we
expect actual clients to do?